src/library_pthread.js

```diff
@@ -331,7 +333,7 @@ var LibraryPThread = {
         worker.on('error', function(e) {
           worker.onerror(e);
         });
-        worker.on('exit', function() {
+        worker.on('detachedExit', function() {
```

Comment:
This should probably be left in place, since the worker never sends the `detachedExit` event. See:
https://nodejs.org/api/worker_threads.html#worker_threads_event_exit

```suggestion
        worker.on('exit', function() {
```

Reply:
Ooope yes, thanks, I was confused between a worker event and the `exit` command we had above.  All the more reason to do the rename or our command.